README.md

```diff
@@ -1,5 +1,9 @@
 # Open Source Help Community
 
+ [![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
+
+
```

Comment:
@Logan1x I can squash while merging on Github but since we want a change in the commit message, you'll have to do it on your own PC. You can follow these steps.

1. `$ git rebase -i HEAD~4`

2. Then change what you get to look like this.

```
pick cbaaa2b the final commit for travis ci bagde
squash 50a1be7 changed the location of badge
squash 4212ed4 The final one
squash 6cd7e32 deleted line breaks

# Rebase f702e46..6cd7e32 onto f702e46 (4 commands)
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
# d, drop = remove commit
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#
# Note that empty commits are commented out
```

This will squash your commits. Save this.

3. Next, git will prompt open another window asking you if you would like to change the commit message. Edit that to look like this:

```
# This is a combination of 4 commits.
# This is the 1st commit message:
README.md: Add Travis CI badge

# This is the commit message #2:
# This is the commit message #3:
# This is the commit message #4:

# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# Author:    Logan1x <sharmakhushal78@gmail.com>
# Date:      Sun Jul 23 18:33:43 2017 +0530
#
# interactive rebase in progress; onto f702e46
# Last commands done (4 commands done):
#    squash 4212ed4 The final one
#    squash 6cd7e32 deleted line breaks
# No commands remaining.
# You are currently editing a commit while rebasing branch 'test' on 'f702e46'.
#
# Changes to be committed:
#       modified:   README.md
#
```

Then you should be good to go!

Please note: Don't copy paste what I pasted above. Edit by hand. Commit hashes may differ, etc. (shouldn't - but better not to risk it :smile: )